deployment/terraform/create.go

```diff
@@ -126,9 +126,15 @@ func (t *Terraform) Create() error {
 	t.setupAppServers(output, extAgent, uploadBinary, binaryPath)
 	// Updating the nginx config on proxy server
 	t.setupProxyServer(output, extAgent)
-	// TODO: display the entire cluster info from terraformOutput later
-	// when we have cluster support.
-	mlog.Info("Deployment complete.")
+	mlog.Info("Deployment complete. Here is the setup information:")
+	mlog.Info("Proxy server: " + output.ProxyIP.Value)
+	mlog.Info("Instances:")
+	for _, instance := range output.Instances.Value {
+		mlog.Info(instance.PublicIP)
+	}
+	mlog.Info("Metrics server: " + output.MetricsServer.Value.PublicIP)
+	mlog.Info("DB reader endpoint: " + output.DBCluster.Value.ReaderEndpoint)
+	mlog.Info("DB cluster endpoint: " + output.DBCluster.Value.ClusterEndpoint)
```

Comment:
Can this become a function that takes `output` and prints the needed info?
This is mainly to keep `Create()` clean and don't clutter it with log calls.

Reply:
Definitely. :+1:

Reply:
Done